app/controllers/stories_controller.rb

```diff
@@ -132,6 +132,9 @@ def handle_tag_index
 
     @num_published_articles = if @tag_model.requires_approval?
                                 Article.published.cached_tagged_by_approval_with(@tag).size
+                              elsif SiteConfig.feed_strategy == "basic"
+                                Article.published.cached_tagged_with(@tag)
+                                  .where("score > #{SiteConfig.tag_feed_minimum_score}").size
```

Comment:
Adding this in to not use the cached value **_at all_** and instead ping the database every time if the feed strategy is basic. I think it's okay for smaller Forems, but maybe we still want to use a cached value and instead change it from 2 hours to something shorter.